pkg/generate/app/cmd/template.go

```diff
@@ -53,7 +53,10 @@ func formatString(out io.Writer, tab, s string) {
 	labelVals := strings.Split(s, "\n")
 
 	for _, lval := range labelVals {
-		fmt.Fprintf(out, fmt.Sprintf("%s%s\n", tab, lval))
+		s := fmt.Sprintf("%s%s\n", tab, lval)
+		if len(strings.TrimSpace(s)) > 0 {
+			fmt.Fprintf(out, fmt.Sprintf("%s%s\n", tab, lval))
+		}
```

Comment:
i don't know that we care *that* much but this is going to strip out newlines in a string like "foo\n\nbar\n" (ie there's an intentional extra newline in the description and this will strip it).

might be better to just strip all trailing newlines from the string via a regex before doing the split.